src/app/downloads/downloads-table.component.css

```diff
@@ -54,6 +54,9 @@ table.scroll-x th:first-child, table.scroll-x td:first-child {
 .table-navigation {
   display: flex;
   justify-content: space-between;
+  padding: 3px;
+  background-color: #fff;
+  margin-top: 3px;
```

Comment:
This is very much just a personal preference, probably should get @farski's opinion as well. I'm not feeling the white background in the table-navigation bar so much. What if the buttons where white with no background on that div?
Instead of:
<img width="909" alt="screen shot 2018-02-14 at 9 06 29 am" src="https://user-images.githubusercontent.com/2250413/36211370-b4356fcc-1166-11e8-9b15-03c010c57c69.png">
we could do this:
<img width="915" alt="screen shot 2018-02-14 at 9 07 46 am" src="https://user-images.githubusercontent.com/2250413/36211391-bf01231a-1166-11e8-947b-9a7f90e3490b.png">

Reply:
It seems like the light grey is reserved for background everywhere else, and that anything clickable is on a white background. It felt "unfinished" somehow to me in that regard. But I'm happy to defer to you and @farski

Reply:
background reverted to grey

![screen shot 2018-02-15 at 12 30 43 pm](https://user-images.githubusercontent.com/1205061/36274131-62d3ce12-124c-11e8-957e-67484e4b479e.png)